src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/model/JudicialProfile.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.orgrolemapping.domain.model;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class JudicialProfile implements Serializable {
+
+    @NonNull
+    private String elinkId;
+    private String personalCode;
+    private String title;
+    private String knownAs;
+    private String surname;
+    private String fullName;
+    private String postNominals;
+
+    private String contractTypeId;
+    private String workPattern;
+    private String emailId;
+    private LocalDateTime joiningDate;
+    private LocalDateTime lastWorkingDate;
+    private String extractedDate;
+    private String activeFlag;
+    private List<Appointment> appointments;
+    private List<Authorisation> authorisations;
+    private String userType;
+
+    @Data
+    @Builder
+    @AllArgsConstructor
+    @NoArgsConstructor
+    @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+    public static class Appointment implements Serializable {
+
+        @NonNull
+        private String roleId;
+        private String roleDescEn;
+        @NonNull
+        private String contractTypeId;
+        private String contractTypeDescEn;
+        @NonNull
+        private String baseLocationId;
+        private String courtName;
+        private String bench;
+        private String courtType;
+        private String circuit;
+        private String areaOfExpertise;
+        @NonNull
+        private String locationId;
+        private String locationDescEn;
+        private String isPrincipalAppointment;
+        @NonNull
+        private LocalDateTime startDate;
+        @NonNull
+        private LocalDateTime endDate;
+        private String activeFlag;
+        private LocalDateTime extractedDate;
+        private String appointmentId;
+    }
```

Comment:
I believe the updated vars for an Appointment are the following:

    private String appointmentId;
    private String roleId;
    private String roleDescEn;
    private String contractTypeId;
    private String contractTypeDescEn;
    private String baseLocationId;
    private String regionId;
    private String regionDescEn;
    private String isPrincipalAppointment;
    private String startDate;
    private String endDate;
    
    Confirm with Abhijit

Reply:
regionId is the missing one

Reply:
Checking and adding.